scripts/export_comments.py

```diff
@@ -43,10 +43,11 @@ def create_comments_excel_file(survey_id, period, submissions):
             continue
         row += 1
         surveys_with_comments_count += 1
-        ws.cell(row, 1, submission.data['metadata']['ru_ref'])
-        ws.cell(row, 2, submission.data['collection']['period'])
-        ws.cell(row, 3, boxes_selected)
-        ws.cell(row, 4, comment)
+        ws.cell(row, 1, submission.data['survey_id'])
```

Comment:
I mentioned to you about whether this is needed.  I doubt it, but they might care which comments came from which surveys

Reply:
I think this should be removed.  If they want it after the fact they can ask for it, but this'll make all the other comment spreadsheets more noisy for the sake of 1 survey.  It doesn't seem worth it.